workshops/serverless-overview/workshop.md

```diff
@@ -14,13 +14,13 @@ contacts: # Required. Must match the number of authors
   - "@justrebl"
   - "@ikhemissi"
 duration_minutes: 180
-tags: azure, azure functions, logic apps, event grid, key vault, cosmos db, csu
+tags: azure, csu, azure functions, logic apps, event grid, key vault, cosmos db, email, app service, web pubsub, static web app
```

Comment:
**Note:** the first 4 tags will be visible and listed in the moaw workshop list.
All tags are searchable though, so usually practical tags like `csu` are hidden towards the end